docs/docusaurus/docs/cloud/quickstarts/python_api_quickstart.md

```diff
@@ -0,0 +1,164 @@
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your user access token and then save the file. 
+
+    GX recommends deleting the temporary file after you set the environment variables.
+
+## Set the GX Cloud Organization ID and user access token as environment variables
+
+Environment variables securely store your GX Cloud access credentials. To use GX Cloud in Python scripts: 
+
+1. Save your **GX_CLOUD_ACCESS_TOKEN** and **GX_CLOUD_ORGANIZATION_ID** as environment variables by entering `export ENV_VAR_NAME=env_var_value` in the terminal or adding the command to your `~/.bashrc` or `~/.zshrc` file. For example:
+
+    ```bash title="Terminal input"
+    export GX_CLOUD_ACCESS_TOKEN=<user_access_token>
+    export GX_CLOUD_ORGANIZATION_ID=<organization_id>
+    ```
+
+    :::note
+    Once you have saved your **GX_CLOUD_ACCESS_TOKEN** and **GX_CLOUD_ORGANIZTION_ID** you will be able to access your GX Cloud account in Python scripts and environments while following the [GX OSS guides](../../oss/oss.md).
+    :::
+
+2. Optional. If you created a temporary file to record your user access token and Organization ID, delete it.
+
+## Create a Data Context
+
+- Use the following command in Python to create a Data Context object:
+
+    ```python name="tutorials/quickstart/quickstart.py get_context"
+    ```
+
+## Connect to data
+
+- Use the following command in Python to connect to existing `.csv` data stored in the `great_expectations` GitHub repository and create a Validator object:
+
+    ```python name="tutorials/quickstart/quickstart.py connect_to_data"
+    ```
+
+    The code example uses the default Data Source for Pandas to access the `.csv` data from the file at the specified URL path.
+
+## Create Expectations
+
+- Use the following commands to create two Expectations and save them to the Expectation Suite:
+
+    ```python name="tutorials/quickstart/quickstart.py create_expectation"
+    ```
+
+  The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null).
+
+  The second Expectation uses explicit kwargs along with the `passenger_count` column.
+
+## Validate data
+
+1. Use the following command to define a Checkpoint and examine the data to determine if it matches the defined Expectations:
+
+    ```python name="tutorials/quickstart/quickstart.py create_checkpoint"
+    ```
+
+2. Use the following command to return the Validation Results:
+
+    ```python name="tutorials/quickstart/quickstart.py run_checkpoint"
+    ```
+
+3. Use the following command to view an HTML representation of the Validation Results in the generated Data Docs:
+
+    ```python name="tutorials/quickstart/quickstart.py view_results"
+    ```
+
+## Related documentation
+
+If you're ready to continue your GX journey, the following topics can help you implement a solution for your specific environment and business requirements:
```

Comment:
```suggestion

```
Don't add unnecessary introductory text. Users understand that the list of topics can be helpful.